### ISSUE TITLE
Overlay-scrollbars list jumps to shorter height in bootstrap-vue popover

### ISSUE THREAD

**Issue by pochern** [KingSora/OverlayScrollbars#368]
**Describe the bug**
Using overlay-scrollbars inside a bootstrap-vue popover (https://bootstrap-vue.org/docs/components/popover) is causing the overlay-scrollbars list to hide before the popover hides. This causes the list to jump shorter right before the popover closes. Is there a way to show the overlay-scrollbars list at the same height until the popover closes? (To avoid the height jump)

**To Reproduce**
Click 'Click for ingredients list' and when the popover is closing you can see it jumps to its min-height: 50px before hiding.

**Expected behavior**
Expected to hide the popover at the same height as when it is showing. Overlay scrollbars list should not jump to shorter in height.

**Examples**
Codesandbox example: https://codesandbox.io/s/wispy-feather-t87b0?file=/src/App.vue

**Environment**
Chrome Version 89.0.4389.114 on macOS Big Sur (same thing on other browsers and OS)

**Comment by KingSora** [comment]
Good day @pochern 

I don't know how the popover is handled but the plugin is destroying in the `beforeDestroy` method (like intended).. Is there any other possibility to destroy it - maybe in a different lifecycle method? (Not an expert in Vue)

**Comment by pochern** [comment]
Thank you so much for your response.

> the plugin is destroying in the beforeDestroy method (like intended)

Is this referring to the overlay-scrollbars?

**Comment by KingSora** [comment]
Yes.. it seems like the popover is doing some animation/transition before it gets completely destroyed, but the `beforeDestroy` lifecycle method is called before that
